app/controllers/users_controller.rb

```diff
@@ -0,0 +1,28 @@
+class UsersController < ApplicationController
+
+  def new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if params[:password] == params[:confirm_password] && @user.save
+      session[:user_id] = @user.id
+      flash[:success] = "You are now registered and logged in"
+      redirect_to '/profile'
+    elsif params[:password] != params[:confirm_password]
```

Comment:
Didn't see a test for this. I will make an issue to create a test for this.

Reply:
Do we want to create a column to to-do's?